### PR TITLE
Update bcm2711_bootloader_config.md

### PR DESCRIPTION
**Pull request by timg236**
* TFTP_TIMEOUT should be TFTP_FILE_TIMEOUT and correct default value
* Add MAC_ADDRESS
* Add USB MSD retries

### CODE REVIEW

File: hardware/raspberrypi/bcm2711_bootloader_config.md [hunk 1]
```diff
@@ -177,6 +177,11 @@ The gateway address to use if the TFTP server is on a differenet subnet e.g. "19
 Default: ""  
 Version: pieeprom-2020-04-16.bin  
 
+#### MAC_ADDRESS
+Overrides the Ethernet MAC address with the given value. e.g. dc:a6:32:01:36:c2`
```
Comment (andrum99) [inline hardware/raspberrypi/bcm2711_bootloader_config.md:181]: There's a stray backtick at the end of this line.

File: hardware/raspberrypi/bcm2711_bootloader_config.md [hunk 2]
```diff
@@ -348,6 +353,12 @@ If no USB mass storage devices are found within this timeout then USB-MSD is sto
 Default: 20000 (20 seconds)  
 Version: pieeprom-2020-05-15.bin - BETA  
 
+### USB_MSD_BOOT_MAX_RETRIES
+The number of times USB MSD boot will be retried after failure before moving to the next boot mode defined by `BOOT_ORDER`.
```
Comment (lurch) [inline hardware/raspberrypi/bcm2711_bootloader_config.md:357]: Does this have the same "-1 means infinite retries" behaviour as the other `_MAX_RETRIES` options?

For consistency, I wonder if the "before moving to the next boot mode defined by `BOOT_ORDER`" text used here should also be added to the descriptions of the other `_MAX_RETRIES` options?

**Comment by lurch** [comment]
P.S. @timg236 It's not directly related to this PR, but I just noticed that the bit about BOOT_ORDER says:
```
0x3 - USB device boot (usbboot)[https://github.com/raspberrypi/usbboot] - Compute Module only.
```
Unfortunately you've got the brackets on your link the wrong way around (a very easy mistake to make!). It should be:
```
0x3 - USB device boot [usbboot](https://github.com/raspberrypi/usbboot) - Compute Module only.
```
If you want to squeeze that into this PR then great, otherwise I'll fix it after this PR has been merged.

File: hardware/raspberrypi/bcm2711_bootloader_config.md (hunk 1 above)
Comment (timg236) [reply hardware/raspberrypi/bcm2711_bootloader_config.md:181]: done

File: hardware/raspberrypi/bcm2711_bootloader_config.md (hunk 2 above)
Comment (timg236) [reply hardware/raspberrypi/bcm2711_bootloader_config.md:357]: ok

**Comment by timg236** [comment]
Thanks all

**Pull request merged**